scripts/vscodeserver/boot_repl.jl

```diff
@@ -0,0 +1,41 @@
+# this script basially only handles `Base.ARGS`
+
+# XXX: is this needed ?
+!(Sys.isunix() || Sys.iswindows()) && error("Unknown operating system.")
+
+# suppress Pkg.jl's messages
+let
+    old = stderr
+    rd, wr = redirect_stderr()
+    err = nothing
+    try
+        using Pkg
+        Pkg.activate(@__DIR__)
+        using vscodeserver
+        Pkg.activate() # back to default env
+    catch e
+        @error e
+    finally
+        redirect_stderr(old)
+        close(wr)
+        close(rd)
+    end
+end
```

Comment:
Could we instead just push a path onto `LOAD_PATH`? If that works we wouldn't have to change the active project at all, which might be more robust?